docs/mp/extensions/04_cdi_oci-objectstorage.adoc

```diff
@@ -18,9 +18,10 @@
 
 = CDI extension for OCI Object storage
```

Comment:
CDI Extension for OCI Object Storage

Reply:
LJ: OCI (Oracle Cloud Infrastructure) spelled out makes the title too long, IMO. I'd leave it OCI here and spell it out in the description below.

Reply:
Done